### PR TITLE
Contact options - showon

### PR DESCRIPTION
**Pull request by brianteeman**
Summary of Changes

Use the showon functionality now available in forms to simplify/reduce the options displayed in the com_contact component options

Testing Instructions

This is only a cosmetic change
Go to the Options page for this component and try all the options on that page to ensure that the new show/hide functionality makes sense.

[Note this is part of a series of PR for each component options page and then the menu options]

### CODE REVIEW

**Comment by Twincarb** [comment]
I have tested this item :white_check_mark: successfully on 4db2490758f5adc078359571166c1cac23cd589f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10153">issues.joomla.org/joomla-cms/10153</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on 4db2490758f5adc078359571166c1cac23cd589f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10153">issues.joomla.org/joomla-cms/10153</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10153">issues.joomla.org/joomla-cms/10153</a>.</sub>

File: administrator/components/com_contact/config.xml [hunk 1]
```diff
@@ -253,7 +255,7 @@
 			type="list"
 			default="10"
 			label="COM_CONTACT_FIELD_ARTICLES_DISPLAY_NUM_LABEL"
-			description="COM_CONTACT_FIELD_ARTICLES_DISPLAY_NUM_DESC"
```
Comment (andrepereiradasilva) [inline administrator/components/com_contact/config.xml]: why remove the description?

Comment (brianteeman) [inline administrator/components/com_contact/config.xml]: Error

**Pull request merged**